examples/imagenet_adv_patch.py

```diff
@@ -0,0 +1,161 @@
+"""A Shift Happens task: Model-aware/agnostic Adversarial Patch Attack on ImageNet"""
+
+import dataclasses
+import os
+
+import numpy as np
+import torch
+import torchvision.datasets as tv_datasets
+import torchvision.transforms as tv_transforms
+import torch.nn as nn
+
+import sys
+sys.path.append('./')
+
+import shifthappens.models.base as sh_models
+import shifthappens.data.base as sh_data
+import shifthappens.data.torch as sh_data_torch
+import shifthappens.utils as sh_utils
```

Comment:
```suggestion
import shifthappens.models.base as sh_models
import shifthappens.data.base as sh_data
import shifthappens.data.torch as sh_data_torch
import shifthappens.utils as sh_utils
```
Should we remove this? These imports are never used.